doc/source/amazon/step-zero-aws.rst

```diff
@@ -7,166 +7,162 @@ AWS does not have native support for Kubernetes, however there are
+  
+       export ZONES=$(aws ec2 describe-availability-zones --region $REGION | grep ZoneName | awk '{print $2}' | tr -d '"')
+       export ZONES=$(echo $ZONES | tr -d " " | rev | cut -c 2- | rev)``
+
+9. Create the cluster
+
+   For a basic setup run the following (All sizes measured in GB)::
+
+       kops create cluster $NAME \
+         --zones $ZONES \
+         --authorization RBAC \
+         --master-size t2.micro \
+         --master-volume-size 10 \
+         --node-size t2.medium \
+         --node-volume-size 10 \
+         --yes
+
+   For a more secure setup add the following params to the kops command::
+   
+         --topology private \
+         --networking weave \
+
+   This creates a cluster where all of the masters and nodes are in private subnets and don't have external IP addresses.  A mis-configured security group or insecure ssh configuration is less likely to comprimize the cluster.
+   In order to SSH into your cluster you will need to set up a bastion node.  Make sure you do that step below.
+   If you have the default number of elastic IPs (10) you may need to put in a request to AWS support to bump up that limit.  The alternative is reducing the number of zones specified.
+   
+   More reading on this subject:
+   https://github.com/kubernetes/kops/blob/master/docs/networking.md
+
+   Settings to consider (not covered in this guide)::
+   
+       --vpc
+         Allows you to use a custom VPC or share a VPC
+         https://github.com/kubernetes/kops/blob/master/docs/run_in_existing_vpc.md
+       --master-count
+         Spawns more masters in one or more VPCs
+         This improves redudancy and reduces downtime during cluster upgrades
+       --master-zones
+         specify zones to run the master in
+       --node-count
+         Increases the total nodes created (default 2)
+       --master/node-security-groups
+         Allows you to specify additional security groups to put the masters and nodes in by default
+       --ssh-access
+         By default SSH access is open to the world (0.0.0.0).
+         If you are using a private topology, this is not a problem.
+         If you are using a public topology make sure your ssh keys are strong and you keep sshd up to date on your cluster's nodes.
+
+10. Wait for the cluster to start-up
+
+    Running the 'kops validate cluster' command will tell us what the current state of setup is.
+    If you see "can not get nodes" initially, just be patient as the cluster can't report until a
+    few basic services are up and running.
+   
+    Keep running 'kops validate cluster' until you see "Your cluster $NAME is ready" at the end of the output.
+   
+    ``time until kops validate cluster; do sleep 15 ; done`` can be used to automate the waiting process.
+    
+    If at any point you wish to destroy your cluster after this step, run ``kops delete cluster $NAME --yes``
+    
+
+11. Confirm that ``kubectl`` is connected to your Kubernetes cluster.
```

Comment:
I have never needed to do those steps?  What does it facilitate?

Reply:
I've done the Cloudformation with Heptio a number of times. 
This is just to give the same functionality as described in steps 3-5 of the Heptio guide.

Part of the reason I bring it up is that it's not the easiest thing to figure out.
But if you copy step 3 from old Heptio. Insert this command, then step 4 and 5 can proceed as described.
This will enable the user later using helm to run that from their own machine rather than copy any customizations and configurations to the host machine.

Reply:
It will integrate this section better with the next section regarding the use of helm

Reply:
Ah ok, our plans are to lock down access to the CI machine used to run the KOPS commands, but this would be good to re-include.  I'll make the change and test on monday when I am back from vacation.

Reply:
I've added and tested e27d5b9